plugins/modules/parcel.py

```diff
@@ -32,11 +32,39 @@
 short_description: Manage the state of parcels on a Cluster
 description:
   - Facilitates the management of parcels on a Cluster by downloading, distributing, and activating them according to the specified state.
-  - Supported states include 'download', 'distribute', and 'activate', each corresponding to specific actions performed on parcels.
 author:
   - "Ronald Suplina (@rsuplina)"
 requirements:
   - cm_client
+options:
+  cluster_name:
+    description:
+      - The name of the cluster
+    type: str
+    required: yes
+  product:
+    description:
+      - The name of the product, e.g. CDH, Impala
+    type: str
+    required: yes
+  version:
```

Comment:
This will conflict with the `version` parameter common to all `cloudera.cluster` modules via the `ClouderaManagerModule` class (and the [`doc_fragment`](https://github.com/cloudera-labs/cloudera.cluster/blob/main/plugins/doc_fragments/cm_options.py#L38-L45))

I would like to change this common parameter to something like `api_version` and allow modules like is one to use `version` as they need. Until then, you will need to "prefix" this parameter, so something like `parcel_version`?